openpaas-james/apps/distributed/README.md

```diff
@@ -0,0 +1,52 @@
+# openpaas-james Distributed server
+
+This server is the distributed version of James relying on:
+
+* Cassandra
```

Comment:
Versions please

Reply:
Ok.... Even if I'm pretty sure those might likely get forgotten to be updated the day we upgrade those